codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -156,14 +175,10 @@ public void sendFlowControlled(ChannelHandlerContext ctx, Http2Stream stream,
         }
         // Save the context. We'll use this later when we write pending bytes.
         this.ctx = ctx;
-        try {
-            FlowState state = state(stream);
-            state.newFrame(payload);
-            state.writeBytes(state.writableWindow());
-            flush();
-        } catch (Throwable e) {
-            payload.error(e);
-        }
+        FlowState state = state(stream);
+        state.newFrame(payload);
+        state.writeBytes(state.writableWindow());
+        flush();
```

Comment:
Understood...I think the majority of cases we are good and perhaps just paranoia but what about RuntimeExceptions, VirtualMachineErrors and the like?  If we want to guarantee our interface it may be prudent to account for these types of situations.  WDYT?